src/translations/en.json

```diff
@@ -130,21 +131,23 @@
   "insuree.ProfessionPicker.label": "Profession",
   "insuree.Profession.null": "Any",
   "insuree.Profession.none": "Not specified",
-  "insuree.Profession.1": "Housewife",
-  "insuree.Profession.2": "Employee",
-  "insuree.Profession.3": "Self Employee",
-  "insuree.Profession.4": "Other",
+  "insuree.Profession.1": "Liberal",
```

Comment:
These terms should be in the language pack.

Reply:
for comores RFC we have others option for profession fields options

Reply:
This is specific to Comores. Please remove this change , they have to manage the correct translation on Lokalize.